### PR TITLE
Feat job: adding getDependenciesCount method

### PR DESCRIPTION
**Pull request by roggervalf**
Would be nice to have a method to get processed and unprocessed counts without bringing all the dependencies values

### CODE REVIEW

File: src/test/test_worker.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { after, times, once } from 'lodash';
 import { RetryErrors } from '../enums';
 import * as sinon from 'sinon';
 
-describe.only('workers', function() {
```
Comment (roggervalf) [inline src/test/test_worker.ts:11]: sorry, my bad

File: src/classes/job.ts [hunk 2]
```diff
@@ -569,8 +571,8 @@ export class Job<T = any, R = any, N extends string = string> {
     const client = await this.queue.client;
     const multi = client.multi();
     if (!opts.processed && !opts.unprocessed) {
-      await multi.hgetall(this.toKey(`${this.id}:processed`));
```
Comment (roggervalf) [inline src/classes/job.ts:572]: these awaits are not necessary

**Comment by ccollie** [comment]
Hey @roggervalf, while you're at it, could you export `DependenciesOpts` ?

https://github.com/taskforcesh/bullmq/blob/87b360323ed3ab4d43ccd3e278413137479cdf3b/src/classes/job.ts#L57

**Comment by roggervalf** [comment]
> Hey @roggervalf, while you're at it, could you export `DependenciesOpts` ?
> 
> https://github.com/taskforcesh/bullmq/blob/87b360323ed3ab4d43ccd3e278413137479cdf3b/src/classes/job.ts#L57

@ccollie, sure no problem

**Comment by roggervalf** [comment]
@manast please when you get a change, some :eyes:

**Comment by manast** [comment]
LGTM. Is this ready for merge?

**Comment by roggervalf** [comment]
@manast yes it is ready

**Comment by manast** [comment]
Great. One little comment though, I noticed that you sometimes use "gerund" verb form in the commits. According to  the semantical commits standard it should be the imperative form of the verb. So instead of "adding" just "add". Keep up with the good work!

**Pull request merged**